ui/StatusQ/src/wallet/managetokenscontroller.cpp

```diff
@@ -227,6 +227,7 @@ void ManageTokensController::savingFinished()
     incRevision();
 
     setSettingsDirty(false);
+    requestLoadSettings();
```

Comment:
@stefandunca I've noticed the updates are not live and changes are seen in other views only after reloading the views. Binding on `revision` is not enough. This is a quick fix to load the backend settings after `save`.

Reply:
Indeed, nice catch.